app/ide-desktop/common/src/text/english.json

```diff
@@ -500,27 +500,27 @@
   "billedMonthly": "Billed monthly",
   "billedAnnually": "Billed annually",
   "tryFree": "Try free for $0 days, then ",
-  "priceTemplate": "$0 per user / month. $1",
+  "priceTemplate": "$0 per user / month, $1",
   "freePlanName": "Free",
   "freePlanSubtitle": "Try Enso",
   "freePlanPricing": "$0 forever",
   "freePlanFeatures": "Interactive Live Data Analytics; Python, Java, JavaScript Support; Local workflow execution",
   "freePlanSeatsDescription": "This plan does not allow to adjust seats",
   "soloPlanName": "Solo",
   "soloPlanSubtitle": "For individuals",
-  "soloPlanPricing": "$60 per user / month. Billed annually",
+  "soloPlanPricing": "$60 per user / month, Billed annually",
```

Comment:
(nit 2: now that the second part is no longer its own sentence, then it shouldn't be capitalized, same with the other two occurrences of "billed annually")